lib/loga/sidekiq/job_logger.rb

```diff
@@ -0,0 +1,59 @@
+module Loga
+  module Sidekiq
+    class JobLogger
+      include Loga::Utilities
+
+      EVENT_TYPE = 'sidekiq'.freeze
+
+      attr_reader :started_at, :data
+
+      def initialize
+        @started_at = Time.now
+        @data = {}
+      end
+
+      def call(item, _queue)
+        yield
+      rescue Exception => ex # rubocop:disable Lint/RescueException
+        data['exception'] = ex
+        data['backtrace'] = caller
```

Comment:
Just passing the exception to the `Loga::Event` should be enough. The formatters will fill the data https://github.com/FundingCircle/loga/blob/master/lib/loga/formatters/gelf_formatter.rb#L83